pages/understanding-json-schema/conventions.md

```diff
@@ -5,11 +5,11 @@ title: "Conventions used in this documentation"
 
 ## Language-specific notes
 
-The names of the basic types in JavaScript and JSON can be confusing
+The names of the basic types in JavaScript and [JSON](../learn/glossary#json) can be confusing
```

Comment:
I'll avoid interlinking JSON because everybody knows it and it is everywhere.

Reply:
Okay I will undo it.